config/application.yml.example

```diff
@@ -48,7 +48,10 @@ queue_adapter_weights: '{"inline": 1}'
 recovery_code_length: '4'
 
 # How long (in seconds) to wait for requests before dropping them (via the rack_timeout gem).
-service_timeout: '15'
+# Note that the key name must be capitalized because the gem looks for
+# ENV['RACK_TIMEOUT_SERVICE_TIMEOUT'], and Figaro does not automatically make lowercase keys
+# available to ENV as uppercase keys.
+RACK_TIMEOUT_SERVICE_TIMEOUT: '15'
```

Comment:
A take it or leave it thought is that we could add an initializer to uppercase this and add it to the env like we did for a little bit with the identity vendors